R/PackageFunctionReporter.R

```diff
@@ -86,13 +86,25 @@ PackageFunctionReporter <- R6::R6Class(
             }
             
             log_info(sprintf('Constructing network representation...'))
-            funcMap <- mvbutils::foodweb(where = paste("package", private$packageName, sep = ":"), plotting = FALSE)
+            # foodweb will output a warning for "In par(oldpar) : calling par(new=TRUE) with no plot" all the time. 
+            # does not seem to be an issue
+            funcMap <- suppressWarnings(mvbutils::foodweb(where = paste("package"
```

Comment:
can you change the indentation to something a bit more readable? Consider

```
funcMap <- suppressWarnings({
    mvbutils::foodweb(
        where = paste("package", private$packageName, sep = ":")
        , plotting = FALSE
    )
})
```

consider it